### PR TITLE
Enable testifylint linter

### PR DESCRIPTION
**Pull request by mmorel-35**
Signed-off-by: Matthieu MOREL <matthieu.morel35@gmail.com>

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -33,6 +34,22 @@ linters-settings:
     local-prefixes: github.com/envoyproxy/go-control-plane
   misspell:
     locale: US
+  testifylint:
+    disable:
+      - compares
+      - float-compare
```
Comment (phlax) [inline .golangci.yml:40]: least indent please

**Comment by phlax** [comment]
cc @alecholmez @jpeach

**Comment by alecholmez** [comment]
I'm good with this change.

**Review by alecholmez — APPROVED** [review]

**Review by phlax — APPROVED** [review]

**Comment by mmorel-35** [comment]
Anything is blocking this to be merged ?

**Review by sunjayBhatia — APPROVED** [review]

**Pull request merged**